src/mechanics/mesh/UnitMeshFem.h

```diff
@@ -23,6 +23,13 @@ MeshFem CreateTriangles(int numX, int numY);
 //! @return created mesh
 MeshFem CreateQuads(int numX, int numY);
```

Comment:
Admittedly outside of this PR, but instead of `CreateTriangles` and `CreateQuads`, i'd prefer `CreateUnitSquare` or `UnitSquareMesh` or something, with the type of element as a parameter.